.github/workflows/main-build.yml

```diff
@@ -0,0 +1,50 @@
+name: Build image and push to ghcr.io
+
+on:
+  workflow_run:
+    workflows: ["Path tests"]
+    types:
+      - completed
+
+jobs:
+  build-and-push:
+    runs-on: "ubuntu-22.04"
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Docker Metadata action
+        id: meta
+        uses: docker/metadata-action@v5
+        env:
+          DOCKER_METADATA_PR_HEAD_SHA: "true"
+        with:
+          images: |
+            ghcr.io/buildwithgrove/path
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=ref,event=tag,suffix=-rc
+            type=sha,suffix=-rc
+            type=sha,format=long,suffix=-rc
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          push: true
```

Comment:
Can we add the tag as a build arg?

eg.
```yaml
- name: Build and push Docker image
  uses: docker/build-push-action@v5
  with:
    push: true
    tags: ${{ steps.meta.outputs.tags }}
    build-args: IMAGE_TAG=${{ steps.meta.outputs.tags }}
....
```

Reply:
This has been addressed.